tests/fixtures/config-extends/error.json

```diff
@@ -1,3 +1,3 @@
 {
-    "extends": "./non-existant.json"
+    "extends": "./non-existent.json"
```

Comment:
This `error.json` test config does not seem to be currently used in this repo, but the change makes sense.